ui-v2/app/routes/dc/kv/create.js

```diff
@@ -20,7 +24,14 @@ export default Route.extend(WithKvActions, {
       create: true,
       isLoading: false,
       item: this.item,
-      parent: repo.findBySlug(key, dc),
+      parent: repo.findBySlug(key, dc).catch(e => {
+        if (e.errors && e.errors[0] && e.errors[0].status == '404') {
+          const url = get(transition, 'intent.url');
+          if (url.endsWith('/create')) {
+            return this.transitionTo('dc.kv.folder', key + '/create/');
```

Comment:
This one is `transitionTo` instead of `replaceWith` - was there a reason for the difference?

Reply:
Think I just missed this, have updated to use `replaceWith` everywhere